ci/deps_parser.py

```diff
@@ -14,6 +14,7 @@
 import os
 import re
 import sys
+import json
```

Comment:
nit: alphabetize imports

Reply:
Thank you for the review, I have updated.